chrome/content/zotero/zoteroPane.js

```diff
@@ -3938,6 +3939,11 @@ var ZoteroPane = new function()
 			}
 		}
 		
+		// Update parent item of notes/attachments
+		if (items.every(item => !item.isRegularItem())) {
```

Comment:
> Maybe future-proof this by specifically checking for `isNote`/`isAttachment`

Good point! Since annotation is not a regular item, this option would appear when annotations are selected, which would not be right.

> Maybe call this “Move to Parent”?

"Move to Parent" sounds good to me!